_posts/2019-07-03-reinforce-2019-recap.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: re:Cap of the AWS re:Inforce 2019
+author: mkainula
+excerpt: Summary of the first ever cloud security focused AWS event held in Boston
+tags:
+- AWS
+- re:Inforce
+---
+
+This year, re:Invent is not the only big official AWS conference organized. Earlier in June re:Mars focusing on machine learning and artificial intelligence was held in Las Vegas and last week a cloud security focused re:Inforce was held in Boston.
+
+Even though the event was not completely sold out, the scale of things was still quite big, with over 8000 participants attending and over 200 breakout sessions to choose from during the two day conference. There were a lot of different types of sessions available for different type of audiences, all the way from CISOs to security engineers and also for software developers.
+
+The event was held in Boston Convention & Exhibition Center, which is a giant expo hall with many levels. In the middle of the hall, you had a "Security Learning Hub" which was basically an exhibition area for partners. Despite the fact that you didn’t need to change venues like in re:Invent, you still had to walk quite a lot during the days and getting from one end of the convention center to the other took some time.
+
+![Security Learning Hub](/img/reinforce-2019-recap/security-learning-hub.jpg)
+
+If you had been to re:Invent before, everything seemed quite familiar. Unfortunately this also applied to the registration process for the sessions. Before the reserved seating registration opened, you had no way to view the sessions you were interested in a calendar view. Luckily [the same handy tool someone had made for re:Invent planning](http://reinvent-planner.aws.carlosesilva.com) also worked for re:Inforce with slight modifications, and because you only had 200 sessions to choose from instead of 2000, fitting the sessions into your calendar was also much easier. Also, you didn't need to worry about if you had enough time to travel between venues during the day, which was a huge plus as now you could actually select the sessions from everything available instead of just the ones located in the venue your previous or next session was going to be, like in re:Invent. Once the registration and reserved seating was done, everything on-site worked smoothly and everything was well organized, just like in re:Invent.
```

Comment:
```Also, you didn't need to worry about whether you had enough time to travel between venues during the day, which was a huge plus as now you could actually select the sessions from everything available instead of just the ones located in the venue your previous or next session was going to be, like in re:Invent.
```
Perhaps a bit too long of a sentence, maybe trim it down a bit?